api/v2/performanceprofile_validation_test.go

```diff
@@ -28,12 +28,23 @@ const (
 
 	//MachineConfigLabelKey defines the MachineConfig label key of the test profile
 	MachineConfigLabelKey = "mcKey"
-	//MachineConfigLabelValue defines the MachineConfig label vlue of the test profile
+	//MachineConfigLabelValue defines the MachineConfig label value of the test profile
 	MachineConfigLabelValue = "mcValue"
 	//MachineConfigPoolLabelKey defines the MachineConfigPool label key of the test profile
 	MachineConfigPoolLabelKey = "mcpKey"
 	//MachineConfigPoolLabelValue defines the MachineConfigPool label value of the test profile
 	MachineConfigPoolLabelValue = "mcpValue"
+
+	//NetDeviceName defines a net device name for the test profile
+	NetDeviceName = "enp0s4"
+	//NetDeviceVendorID defines a net device vendor ID for the test profile
+	NetDeviceVendorID = "0x1af4"
+	//NetDeviceModelID defines a net device model ID for the test profile
+	NetDeviceModelID = "0x1000"
+	//NetDeviceMAC defines a net device PCI path for the test profile
+	NetDevicePCIpath = "pci-0000:00:04.0"
+	//NetDeviceMAC defines a net device MAC address for the test profile
+	NetDeviceMAC = "enx54ee75491111"
```

Comment:
shouldn't MAC look like `52:54:00:0b:b7:fb`?

Reply:
We  can match anything udev knows about the device - e.g. udevadm info -p <device path>
The format presented here `ID_NET_NAME_MAC=prefixxAABBCCDDEEFF` matches the way udev maps this MAC address.
What I will probably add will be matching it as a regular expression which means we can e.g. use:
ID_NET_NAME_MAC=.*xAABBCCDDEEFF

I will probably add a validation to accept valid MAC addresses and add an internal functions for conversions from the EUI-48/64  for more simple usage in the tuned base profile we have, e.g.:
[net]
devices_udev_regex=${f:net_eui64_to_udev_regex:${EUI_64_ADDRESS}}
(OR do the translation on our side but the functions already exit in tuned side)

Reply:
btw, as you also wrote, users will likely expect the simplest format I mentioned, so maybe let's start with this?

Reply:
Yeah, using the basic format is probably the correct way to do this. Most of the tools show it like that (ip a, ifconfig..).